src/components/SharedComponents/SubjectPhasePicker/SubjectPhasePicker.tsx

```diff
@@ -180,6 +187,109 @@ const SelectionDropDownBox = styled(Box)<object>`
   box-shadow: 0px 8px 8px 0px rgba(92, 92, 92, 0.2);
 `;
 
+type SubjectContainerProps = {
+  children: React.ReactNode;
+  showSubjectError: boolean;
+  onClick: () => void;
+};
+function SubjectContainer({
+  children,
+  showSubjectError,
+  onClick,
+}: SubjectContainerProps) {
+  const subjectErrorId = useId();
+  const subjectInputId = useId();
+  const isMobile = useMediaQuery("mobile");
+
+  return (
+    <Box
+      $maxHeight={["calc(100vh - 160px)", "auto"]}
```

Comment:
Since this is quite specialised now, rather than adding an additional type in https://github.com/oaknational/Oak-Web-Application/pull/2991/files#diff-98ee9ade28de4475b0c6b48173ce722b73bbc912e588c2fb751e8e2da0cfd117R139 maybe we can instead wrap in another `<div/>` or can we use flex to fill the vertical space.